src/explore-education-statistics-common/src/components/form/__tests__/FormComboBox.test.tsx

```diff
@@ -173,632 +179,552 @@ describe('FormComboBox', () => {
       expect(option2).toHaveAttribute('aria-selected', 'false');
       expect(option3).toHaveAttribute('aria-selected', 'true');
     });
-  });
-
-  describe('list box', () => {
-    test('pressing ArrowUp will cycle the selected item through the entire list', () => {
-      const { container, getByText, getByLabelText } = render(
-        <FormComboBox
-          id="test-combobox"
-          inputLabel="Choose option"
-          onInputChange={() => {}}
-          onSelect={() => {}}
-          options={['Option 1', 'Option 2', 'Option 3']}
-        />,
-      );
-
-      const input = getByLabelText('Choose option') as HTMLInputElement;
-
-      fireEvent.change(input, {
-        target: {
-          value: 'Test',
-        },
+    describe('list box', () => {
+      test('pressing ArrowDown will cycle the selected item through the entire list', () => {
+        const { container, getByText, getByLabelText } = render(
+          <FormComboBox
+            id="test-combobox"
+            inputLabel="Choose option"
+            onInputChange={noop}
+            onSelect={noop}
+            options={['Option 1', 'Option 2', 'Option 3']}
+          />,
+        );
+
+        const input = getByLabelText('Choose option') as HTMLInputElement;
```

Comment:
If we refactor `fireEvent.change` to `userEvent.type` below, pretty sure we don't need a type assertion that this is a `HTMLInputElement` anymore. We could simplify this code to something like:

```ts
await userEvent.type(screen.getByLabelText('Choose option'), 'Test');
```

Same would apply to other test cases where we're doing this.